notebooks/224-vision-image-colorization/224-vision-image-colorization.ipynb

```diff
@@ -0,0 +1,511 @@
+    "* Model was trained on ImageNet dataset.\n",
+    "* Model consumes as input L-channel of LAB-image and give as output predict A- and B-channels of LAB-image."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ae449c6-8a58-4d2c-8389-fde359ad7d1a",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "dd439f78-a4b6-48b2-946e-a308383cf787",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/art3mis/miniconda3/envs/openvino_env/lib/python3.7/site-packages/numpy/__init__.py:156: UserWarning: mkl-service package failed to import, therefore Intel(R) MKL initialization ensuring its correct out-of-the box operation under condition when Gnu OpenMP had already been loaded by Python process is not assured. Please install mkl-service package, see http://github.com/IntelPython/mkl-service\n",
+      "  from . import _distributor_init\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cv2\n",
+    "import numpy as np\n",
+    "from typing import Dict, Tuple\n",
+    "import matplotlib.pyplot as plt\n",
+    "from openvino.runtime import Core"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72d6ca64-aa95-4e50-b5a6-975259d16dcc",
+   "metadata": {},
+   "source": [
+    "## Configurations\n",
+    "\n",
+    "* `PRECISION` - {FP16, FP32}, default: FP16 \n",
+    "* `MODEL_DIR` - directory where the model is to be stored, default: public.\n",
+    "* `MODEL_NAME` - name of the model used for inference, default: colorization-v2\n",
+    "* `DATA_DIR` - directory where test images are stored, default: data\n",
+    "* `DEVICE` - {CPU, GPU, GNA,VPU} device to used for inference, default: CPU"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "02dc1b06-b3ae-40f2-8fca-1368455a369c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "PRECISION = \"FP16\"\n",
+    "MODEL_DIR=\"public\"\n",
+    "MODEL_NAME=\"colorization-v2\"\n",
+    "MODEL_PATH = f\"{MODEL_DIR}/colorization-v2/{PRECISION}/{MODEL_NAME}.xml\"\n",
+    "DATA_DIR = \"data\"\n",
+    "DEVICE=\"CPU\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cf87170-fd22-4fcf-b687-51b404be0db4",
+   "metadata": {},
+   "source": [
+    "## Download the model\n",
+    "\n",
+    "`omz_downloader` downloads model files from online sources and, if necessary, patches them to make them more usable with Model Convertor.\n",
+    "\n",
+    "In our case `omz_downloader` downloads the checkpoint and pytorch model of [colorization-v2](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/colorization-v2/README.md) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and saves it under `MODEL_DIR` as specified in above configuration"
```

Comment:
Avoid the use of bash commands. Instead of using `!omz_downloader --name colorization-v2` use something like this:
````
base_model_dir = "model"

model_name = "colorization-v2" #"colorization-siggraph"

download_command = f"omz_downloader " \
                   f"--name {model_name} " \
                   f"--output_dir {base_model_dir} " \
                   f"--cache_dir {base_model_dir}"
! $download_command
```

Reply:
![image](https://user-images.githubusercontent.com/10940214/165587975-a6f60896-3f4a-46df-832b-8c6539101853.png)